testing/mozjs68/APKBUILD

```diff
@@ -0,0 +1,114 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=mozjs68
+pkgver=68.2.0
+_majver=${pkgver%%.*}
+pkgrel=0
+pkgdesc="Standalone Mozilla JavaScript engine"
+url="https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey"
+arch="x86_64 armv7 armhf aarch64 x86 ppc64le" # limited by rust and cargo
+license="MPL-2.0"
+depends_dev="
+	icu-dev
+	nspr-dev
+	libffi-dev
+	readline-dev
+	"
+makedepends="
+	$depends_dev
+	zlib-dev
+	python2
+	python3-dev
+	perl
+	sed
+	autoconf2.13
+	linux-headers
+	rust
+	cargo
+	llvm9
+	clang-dev
+	"
+subpackages="$pkgname-dev"
+source="https://ftp.mozilla.org/pub/firefox/releases/${pkgver}esr/source/firefox-${pkgver}esr.source.tar.xz
+	0001-silence-sandbox-violations.patch
+	fix-musl-build.patch
+	fix-rust-target.patch
+	"
+builddir="$srcdir"/firefox-$pkgver
+_builddir="$builddir/js/src"
+
+# secfixes:
+#   60.7.2-r0:
+#     - CVE-2019-11708
+#     - CVE-2019-11707
+
+build() {
+	cd "$_builddir"
+
+	export LDFLAGS="$LDFLAGS -Wl,-z,stack-size=1048576"
+
+	# avoid complains about autoconf
+	touch configure
+
+	# Fix broken ARM_ARCH check which doesn't work with newer GCCs it seems.
+	case "$CTARGET" in
+		armv*)
+			ARM_ARCH=$(echo ${CTARGET#*armv} | sed "s|-.*||")
+			sed -i "s|ARM_ARCH=.*|ARM_ARCH=$ARM_ARCH|" "$builddir"/build/autoconf/arch.m4
+			;;
+	esac
+
+	case "$CARCH" in
+		x86)
+			export RUST_TARGET="i686-unknown-linux-musl"
+			;;
+		x86_64)
+			export RUST_TARGET="$CTARGET"
+			;;
+		aarch64)
+			export RUST_TARGET="aarch64-unknown-linux-musl"
+			;;
+		armv7)
+			export RUST_TARGET="armv7-unknown-linux-musleabihf"
+			;;
+		armhf)
+			export RUST_TARGET="arm-unknown-linux-musleabihf"
+			;;
+		ppc64le)
+			export RUST_TARGET="powerpc64le-unknown-linux-musl"
+			;;
+	esac
```

Comment:
I think you're using an older version of the fix-rust-target.patch where you don't have to export RUST_TARGET

Reply:
Done. But I still messing with paths